config/routes.rb

```diff
@@ -11,7 +11,7 @@
   post '/login', :to => 'sessions#create'
   delete '/logout', :to => 'sessions#destroy'
 
-
+  resources :charges
```

Comment:
definitely want to make sure we limit the actions here. I'll try to do that when I pull this version